treeherder/model/management/commands/load_initial_data.py

```diff
@@ -11,4 +11,5 @@ def handle(self, *args, **options):
                      'repository',
                      'failure_classification',
                      'performance_framework',
-                     'performance_bug_templates')
+                     'performance_bug_templates',
+                     'load_preseed')
```

Comment:
This change adds "load_preseed" to the list of fixtures loaded - the later parameters in this list are additional arguments to the `loaddata` command:
https://docs.djangoproject.com/en/1.9/ref/django-admin/#loaddata

However preseed.json isn't a fixture in the same sense as the existing ones (it needs pre-processing to remove the wildcards from what I can tell), so I believe it still needs to be run standalone from loaddata, via the `load_preseed` command you already have. 

eg

```python
        call_command('loaddata',
                     'repository_group',
                     'repository',
                     'failure_classification',
                     'performance_framework',
                     'performance_bug_templates')
        call_command('load_preseed')
```